src/fuzz_introspector/frontends/frontend_cpp.py

```diff
@@ -16,50 +16,28 @@
 
 from typing import Any, Optional
 
+from tree_sitter import Language, Node
+
 import os
 import logging
-
-from tree_sitter import Language, Parser, Node
-import tree_sitter_cpp
 import yaml
 
-from fuzz_introspector.frontends.datatypes import Project
+from fuzz_introspector.frontends.datatypes import Project, SourceCodeFile
 
 logger = logging.getLogger(name=__name__)
 
 
-class SourceCodeFile():
+class CppSourceCodeFile(SourceCodeFile):
```

Comment:
I prefer to have the language post `SourceCodeFile` here, this applies to the other classes as well.